python/py-wmctrl/Portfile

```diff
@@ -0,0 +1,26 @@
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-wmctrl
+version             0.3
+platforms           darwin
```

Comment:
please add ```supported_archs noarch``` under the ```platforms``` field as the port does not install any architecture-dependent files.

Reply:
Done